app/custom_auth.py

```diff
@@ -0,0 +1,89 @@
+from django.contrib import auth
```

Comment:
Consider running [black](https://github.com/python/black) over this file to ensure that the code style is fine, e.g. things like space after comma, two newlines between functions, etc.